request-management-api/request_api/models/FOIRawRequests.py

```diff
@@ -27,15 +32,28 @@ def saverawrequest(cls,_requestrawdata)->DefaultMethodResult:
         db.session.commit()               
         return DefaultMethodResult(True,'Request added',newrawrequest.requestid)
 
+    @classmethod
+    def saverawrequestversion(cls,_requestrawdata,requestid,assignee,status)->DefaultMethodResult:        
```

Comment:
Looks good!  Should be safe from SQL injection too, while still dropping down to the query level.